### PR TITLE
Upgrade project dependencies

### PR DESCRIPTION
**Pull request by rock3r**
Hey there, I am planning to build upon Wien for a project of mine, and will likely contribute some stuff on top of the repo. I started by upgrading the project dependencies, then I have another PR ready that does some minor cleanups (mostly automated and aesthetic stuff, respecting the existing codestyle).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -7,12 +7,120 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+max_line_length = 150
```
Comment (rock3r) [inline .editorconfig:10]: Most of these things are simply the result of exporting the code style settings from IJ as they were already used in the project (essentially, the Kotlin official style)

**Comment by rock3r** [comment]
You may have some config issues with coverage setup — there's no functional changes in the PR. Coverage has not effectively changed.

image

**Comment by mpgirro** [comment]
Hey there! Thank you for your interrest in this project. I'm looking forward to your contributions. 

The PRs look good, I'll merge them right away. The coverage issue is odd, I'll have a look later.

There is one other thing you should be aware of: I am planning to change the namespace of this project before publishing an artifact, because I deprecated the domain behind the namespace a while ago. Should have done this sooner, but it became a new year's resolution ;-)

Let me know when there is a good time to do this, so there will be minimal interference with the contributions you are currently working on.

**Comment by rock3r** [comment]
Thanks @mpgirro — I have another big chunk of work that is almost done and will PR soon. I have reorganised a bit the builder and parser namespaces, extracted interfaces for builders (so that I can have fakes for tests), improved/added many tests. I have also tweaked the models to ensure mandatory fields are non-nullable, according to the various NS specs (where available).

The next step is not entirely pushed yet, but should be in a matter of minutes. Once #6 is merged, I will open a PR for that, but you can already see a preview of the changes [here](https://github.com/rock3r/wien/compare/cleanup_code...rock3r:fix_parsers).

I understand it's a big change coming in, and will understand if you decide not to take it in, but I need to add a few more features to the library (including supporting the `podcast:` namespace, and writing an RSS feed back from the data model) for the project I want to use it for, and I couldn't otherwise.

**Comment by rock3r** [comment]
Update: all the aforementioned changes are pushed. I am adding tests for the `Validating*Builder` classes, but those will be part of a separate PR, as adding support for `podcast:`. The writing RSS part will be done after all this. I have some prototype code for the writer already, but will need to rebase & fixup after the next PR two PRs are merged. If you want to change the namespace, you can do it after I'm done with the writer? Should be a few days' work, tops. Is that ok for you?

**Comment by mpgirro** [comment]
Writers and the `podcast:` NS coming in sounds great! Don't worry about them being included.

Namespace timeline also sounds fine for me, just let me know when you are good to go.

**Pull request merged**